CRM/Financial/Form/FinancialType.php

```diff
@@ -120,7 +120,10 @@ public function postProcess() {
       foreach (['is_active', 'is_reserved', 'is_deductible'] as $field) {
         $params[$field] ??= FALSE;
       }
-      $financialType = civicrm_api3('FinancialType', 'create', $params);
+      $financialType = civicrm_api4('FinancialType', 'create', [
```

Comment:
Oh, now the problem is if you edit one it fails with get "Cannot pass id to create action"

Reply:
Gotta use the `save` action; it's ok with implied-update-by-passing-the-id.